### ISSUE TITLE
Does not log in to accounts

### ISSUE THREAD

**Issue by rmka** [Marc3842h/Titan#62]
After pressing report button the following console messages appears:
https://i.imgur.com/BXLu8yS.png
and then nothing happens, except that quartz scheduler thing
Accounts is ok, worked fine with another reportbot

Win7 x64
Titan 1.6.0.0 built from source today

**Comment by raspbianlike** [comment]
Yeah probably works now, if the issue persists, please open a new issue
